### PR TITLE
Add a function to use the complement

### PR DESCRIPTION
**Pull request by sloriot**
Small feature: [here](https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/Small_Features/FFG_invert_selection)

Link to the doc: [here](https://cgal.github.io/6065/v1/BGL/structCGAL_1_1Face__filtered__graph.html#aa428541ebbdd35f9a6e9a3ffd60178df)

Pre-approved on 2/4/22 by @afabri

### CODE REVIEW

**Comment by afabri** [comment]
It would be good to add a description at the top. Will that be documented, become a small feature....

**Comment by lrineau** [comment]
> It would be good to add a description at the top. Will that be documented, become a small feature....

@sloriot: Do you agree? That is still marked as TODO.

**Comment by sloriot** [comment]
Small feature created cc @afabri

**Comment by lrineau** [comment]
Can you please resolve the conflict, and re-test the pull-request? It was marked https://github.com/CGAL/cgal/labels/Tested more than two months ago.

File: Installation/CHANGES.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ Release date: June 2022
 
 -   Added an overload of the function `CGAL::convex_hull_3()`, which writes the result in an indexed triangle set.
 
+### [CGAL and the Boost Graph Library (BGL)](https://doc.cgal.org/5.5/Manual/packages.html#PkgBGL)
+
+-   Added the function [`invert_selection()`](https://doc.cgal.org/5.5/BGL/structCGAL_1_1Face__filtered__graph.html#aa428541ebbdd35f9a6e9a3ffd60178df) in the class [`Face_filtered_graph`](https://doc.cgal.org/5.5/BGL/structCGAL_1_1Face__filtered__graph.html) to unselect selected faces
+    and to select unselected one.
```
Comment (MaelRL) [inline Installation/CHANGES.md:21]:
```suggestion
-   Added the function [`invert_selection()`](https://doc.cgal.org/5.5/BGL/structCGAL_1_1Face__filtered__graph.html#aa428541ebbdd35f9a6e9a3ffd60178df) in the class [`Face_filtered_graph`](https://doc.cgal.org/5.5/BGL/structCGAL_1_1Face__filtered__graph.html), which toggles the selected status of a graph: selected faces are deselected, and unselected faces are selected.
```

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.5-Ic-27

**Pull request merged**